src/components/SpaceProposalBoost.vue

```diff
@@ -46,14 +46,24 @@ const newBoostLink = computed(() => ({
 const isActive = computed(() => props.proposal.state === 'active');
 const isFinal = computed(() => props.proposal.scores_state === 'final');
 
+const winningChoice = computed(() => {
+  const maxScore = Math.max(...props.proposal.scores);
+  const maxScoreCount = props.proposal.scores.filter(
+    score => score === maxScore
+  ).length;
+  return maxScoreCount > 1 ? 0 : props.proposal.scores.indexOf(maxScore) + 1;
```

Comment:
What is this function returning ?

From the usage below, it seems it returns the `choice` index, starting from 1, but this function is also returning `0` in case there's a draw, which will make nobody win in case of draw. 

Is that correct ?